unity/ARUI/Readme_ARUI.txt

```diff
@@ -0,0 +1,52 @@
+How to use ARUI in your scene:
+
+* Create empty GameObject in the highest hierarchy level
+* Add the AngelARUI script to it. (The script generates all necessary UI elements at run time)	
+
+**** Examples
+
+*** Set task graph:
+For now the ARUI supports a two level task graph: 
+To set the task graph call:
+
+AngelARUI.Instance.SetTasks(tasks);
+
+where 'tasks' could look like this:
+    string[,] tasks =
+    {
+        {"0", "Text example MainTask 1"},
+        {"1", "Text example Subtask 1 of MainTask 1"},
+        {"1", "Text example Subtask 2 of MainTask 1"},
+	{"1", "Text example Subtask 2 of MainTask 1"},
+	{"1", "Text example Subtask 2 of MainTask 1"},
+        {"0", "Text example MainTask 2"},
+        {"0", "Text example MainTask 3"},
+	{"1", "Text example Subtask 1 of MainTask 3"},
+	{"1", "Text example Subtask 2 of MainTask 3"},
```

Comment:
It's not a blocker at the moment, but there is a mixture of tabs and spaces here (and likely in other locations). For consistency in viewing across editing environments (and web viewers like here!), the general practice is to prefer spaces over tabs.